frontend/packages/ux-editor/src/components/config/EditFormComponent.tsx

```diff
@@ -77,7 +76,13 @@ export const EditFormComponent = ({
     return <UnknownComponentAlert componentName={component.type} />;
   }
 
-  return (
+  return component.type === ComponentType.RepeatingGroup ? (
+    <RepeatingGroupComponent
```

Comment:
I suggest that we now move this into the `ComponentSpecificConfig`-component, and remove this check entirely. Since you have enabled the schema for RepeatingGroup in this PR as well, it will work like the other components with custom config and show the custom stuff when beta-mode is disabled 😊

Reply:
Good point 😉 Will fix!